lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -0,0 +1,113 @@
+defmodule Mix.Tasks.Escript.Install do
+  use Mix.Task
+
+  @shortdoc "Install an escript locally"
```

Comment:
I have extracted some pieces of functionality used by `archive` and `archive.uninstall` into `Mix.Local.Utils`. However, the installation processes for archives and escripts are quite different, I didn't try to unify them.